server/controllers/auth/authentication.js

```diff
@@ -0,0 +1,9 @@
+const { jwtVerify } = require('../../helpers/jwtVerifyPromise');
+
+const key = process.env.KEY;
+exports.auth = (req, res, next) => {
+  const { token } = req.cookies;
+  jwtVerify(token, key)
+    .then((payload) => { req.user = payload; })
+    .then(() => next());
```

Comment:
```suggestion
    .then(next);
```

Reply:
because if we pass anything to the function
it will direct us to the server error handler
so here we are making sure we are passing nothing to it goes to the next middleware
@11janci